pennylane/transforms/core/transform.py

```diff
@@ -217,6 +222,50 @@ def qnode_circuit(a):
           and a transformed :class:`pennylane.devices.Device` is returned. The transform is added
           to the end of the device program and will be last in the overall transform program.
 
+    .. details::
+        :title: Transforms with Catalyst and MLIR
+
+        Transforms can be matched with an associated MLIR or XDSL pass via the ``pass_name`` argument.
+        These transforms will be lowered into the MLIR and applied as part of the lower level compilation.
```

Comment:
```suggestion
        If a transform exists in the Catalyst frontend and *not* in the PennyLane frontend, using the Catalyst transform
        on a `qjit`'d workflow with capture enabled (:func:`pennylane.capture.enable`) requires that it be registered in PennyLane via the ``pass_name``
        argument. These transforms will be lowered into the MLIR and applied as part of the lower-level compilation.
```

Reply:
I'm having a hard time figuring out what this means.

Reply:
That is incorrect.

We are not "registering in PennyLane". We are creating a transform that can be lowered to an MLIR or XDSL pass.

Reply:
```suggestion
        If a compilation pass exists in Catalyst or is implemented as a third-party plugin, using it on a ``qjit``'d workflow requires  that a transform 
        be created in PennyLane with the same name, denoted by the ``pass_name`` argument. This ensures that the transform is properly applied as 
        part of the lower-level compilation. 
```
Removed the capture sentence 👀. 

> For "with the same name", what exactly does that mean? For me, that sounds like we care about the name of the object, which is not the case.

Is there ever a case where we'd recommend using a different name than the Catalyst transform function nam? 🤔